src/wasm/request.rs

```diff
@@ -172,6 +172,16 @@ impl RequestBuilder {
         self
     }
 
+    /// Enable HTTP bearer authentication.
+    pub fn bearer_auth<T>(self, token: T) -> RequestBuilder
```

Comment:
This seems pretty simple to just have in a separate PR that we can merge quickly.